src/Symfony/Component/PasswordHasher/Hasher/UserPasswordHasherInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PasswordHasher\Hasher;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Interface for the user password hasher service.
+ *
+ * @author Ariel Ferrandini <arielferrandini@gmail.com>
+ */
+interface UserPasswordHasherInterface
+{
+    /**
+     * Hashes the plain password.
```

Comment:
```suggestion
     * Hashes the plain password based on the user.
```

(maybe explicitly explain that this does not *set* the password on the user? not sure)